matching-notification-api/src/main/resources/ApplicationResources.properties

```diff
@@ -172,6 +172,10 @@ phenotips.myMatches.contact.dialog.to.label=To
 phenotips.myMatches.contact.dialog.from.label=From
 phenotips.myMatches.contact.dialog.subject.label=Subject
 phenotips.myMatches.contact.dialog.message.label=Message
+phenotips.myMatches.contact.dialog.selectPatientToContactTitle=Select which of the patients to contact
+phenotips.myMatches.contact.dialog.selectPatientToContactText=Please select which patient to contact
```

Comment:
Nobody is contacting patients. Some possible suggestions:
- replace "patient" with "patient record owner"
- use "person of contact" , but then we'd have to use a different verb instead of "contact"
- rephrase the whole thing as a question: "who should be contacted"/"whom to contact"